tests/integration/ci-runner.py

```diff
@@ -314,13 +319,14 @@ def _pre_pull_images(self, repo_path):
                 )
                 return
             except subprocess.CalledProcessError as err:
-                logging.info("docker-compose pull failed: " + str(err))
+                logging.info("docker-compose pull failed: %s", str(err))
```

Comment:
`str(err)` here and everywhere else is redundant. Each parameter in `logging.info` and similar methods are translated into `str` explicitly later. So we can save some unnecessary actions by passing just `err`, without converting into strings